PendingReleaseNotes

```diff
@@ -9,6 +9,15 @@
   recommended running this command against every bucket in every zone after all
   zones have upgraded.
 
+* mgr/volumes: The `subvolume snapshot clone` command now depends on config option
+  `snapshot_clone_no_wait` which is used to reject the clone operation when
+  the cloner threads are not available.It is enabled by default, which means that if
```

Comment:
nit: ... when all cloner threads are busy. This config is enabled by default.

Reply:
done